inputs/tcp/tcp.go

```diff
@@ -45,34 +45,23 @@ func (l *TcpInput) Init(config inputs.MothershipConfig) error {
 }
 
 func (l *TcpInput) Run(output chan common.MapStr) error {
-  logp.Debug("tcpinput", "Running TCP Input")
+  logp.Info("[TcpInput] Running TCP Input")
```

Comment:
I think I'd prefer these stay debug, you can see them with "-d tcpinput" (note, in debug case, not TcpInput)